src/main/java/org/iitbact/cc/FirebaseConfig.java

```diff
@@ -25,7 +25,7 @@
     @Value("${firebase.dburl}")
     private String dbUrl;
 
-    @Value("${env}")
+    @Value("${ENV}")
```

Comment:
We will need to expose ENV in application.properties. It is not present as of now.

Reply:
Put it in system variables ENV=dev
It's written in readme